composables/analytics.js

```diff
@@ -0,0 +1,330 @@
+import dayjs from 'dayjs'
+
+// note: build step can miss unix import for some reason, so
+// we have to import it like this
+// eslint-disable-next-line import/no-named-as-default-member
+const { unix } = dayjs
+
+export const countryCodeToName = (code) => {
+  const regionNames = new Intl.DisplayNames(['en'], { type: 'region' })
+  return regionNames.of(code) || 'Unknown'
+}
```

Comment:
VIntl alternative which would use the user's language.

```js
export function useCountryNames(style = 'long') {
  const formattingOptions = { type: 'region', style }
  const { formats } = useVIntl()
  return function formatCountryName(code) {
    return formats.displayName(code, formattingOptions)
  }
}
```

<details><summary>TypeScript</summary>
<p>

```ts
export function useCountryNames(style: 'narrow' | 'short' | 'long' = 'long') {
  const formattingOptions = { type: 'region', style }
  const { formats } = useVIntl()
  return function formatCountryName(code: string) {
    return formats.displayName(code, formattingOptions)
  }
}
```

</p>
</details> 

Usage:
```ts
const formatCountryName = useCountryNames()
const taiwan = computed(() => formatCountryName('TW'))
```

[💠 Live demo](https://stackblitz.com/edit/vitejs-vite-jpk2bz?file=src%2FApp.vue)